src/commands/shared.ts

```diff
@@ -90,6 +90,7 @@ export const withWalletOption = (yargs: Argv): Argv =>
     type: "string",
     description: "Path to file containing private key of owner account",
     normalize: true,
+    demandOption: true,
```

Comment:
hmm this will break things (we can pass in also pass in pte key right?), y do u need to it to be mandatory?

Reply:
i see currently only config create uses it, but still i need it to be a required field, so i made it required. The description a bit wrong tho. i updated it.